### PR TITLE
Belgien, finnland

### PR DESCRIPTION
**Pull request by dsarkar**
closes https://github.com/corona-warn-app/cwa-website/issues/754

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -342,8 +342,8 @@
...
+                            "(Status: 6.01.2021)",
```
Comment (Ein-Tim) [inline src/data/faq.json]:
```suggestion
                            "(Status: 06.01.2021)",
```

File: src/data/faq_de.json [hunk 2]
```diff
@@ -351,8 +351,8 @@
...
+                            "(Stand: 6.01.2021)",
```
Comment (Ein-Tim) [inline src/data/faq_de.json]:
```suggestion
                            "(Stand: 06.01.2021)",
```

**Comment by Ein-Tim** [comment]
I left to suggestion, if we write the month 01 and not 1 we should also write the day 06 and not 6. 👍

**Comment by dsarkar** [comment]
EN

image

image


DE

image


image

**Review by loki-cui — CHANGES_REQUESTED** [review]
From what I understood is that Belgium might no go live today, so I propose to put it on "pending" until it's final.

**Review by Ein-Tim — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim Ah... could have just accepted your commits.... next time. Thanks anyway.

**Comment by dsarkar** [comment]
@loki-cui ok. blocked until further notice.

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json]: ok

**Comment by MikeMcC399** [comment]
The English dates on
/en/faq/#interoperability_countries
are not formatted with an English locale, and this is generally so on the https://www.coronawarn.app/ site.

In UK the date would be written 06/01/2021 with "/" delimiters instead of ".".
I actually prefer writing the date completely differently as Jan 6, 2021 in English because using the "/" notification you often don't know if it is an American format mm/dd/yyyy or a UK format dd/mm/yyyy with month and day interchanged. Another way which is unambiguous is 06-Jan-2021. That works quite well for German as well and a lot of the months have the same first three characters in English and German. Jan, Feb, (Mar/Mär), Apr, (May, Mai), Jun, Jul, Aug, Sep, (Oct/Okt), Nov, (Dec/Dez).

**Comment by dsarkar** [comment]
@MikeMcC399 I see, next time we will change it. Thanks.

**Pull request merged**